src/config/settings.py

```diff
@@ -182,3 +184,6 @@
 SWAGGER_SETTINGS = {
     "DEFAULT_INFO": "config.swagger_info.SAFE_CONFIG_SERVICE_SWAGGER_INFO"
 }
+
+CORS_ALLOW_ALL_ORIGINS = True
```

Comment:
@Uxio0 Thoughts on this one? This will set the header `Access-Control-Allow-Origin: *`

Reply:
Same is done in the other services

See https://github.com/gnosis/safe-transaction-service/blob/master/config/settings/base.py#L161 and https://github.com/gnosis/safe-client-gateway/blob/main/src/utils/cors.rs#L21

Reply:
As @rmeissner said, we are using it as it's required if you don't want to be whitelisting client by client